interface/ceed-basis.c

```diff
@@ -323,6 +323,47 @@ int CeedBasisGetCollocatedGrad(CeedBasis basis, CeedScalar *collo_grad_1d) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Return 1D interpolation matrix to Chebyshev polynomial coefficients on quadrature space
+
+  @param[in]  basis               `CeedBasis`
+  @param[out] chebyshev_interp_1d Row-major (`P_1d * Q_1d`) matrix interpolating from basis nodes to Chebyshev polynomial coefficients
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
+**/
+int CeedBasisGetChebyshevInterp1D(CeedBasis basis, CeedScalar *chebyshev_interp_1d) {
+  CeedInt           P_1d, Q_1d;
+  CeedScalar       *C, *chebyshev_coeffs_1d_inv;
+  const CeedScalar *interp_1d, *q_ref_1d;
+  Ceed              ceed;
+
+  CeedCall(CeedBasisGetCeed(basis, &ceed));
+  CeedCall(CeedBasisGetNumNodes1D(basis, &P_1d));
+  CeedCall(CeedBasisGetNumQuadraturePoints1D(basis, &Q_1d));
+
+  // Build coefficient matrix
+  // -- Note: Clang-tidy needs this check because it does not understand the is_tensor_basis check above
+  CeedCheck(P_1d > 0 && Q_1d > 0, ceed, CEED_ERROR_INCOMPATIBLE, "CeedBasis dimensions are malformed");
```

Comment:
```suggestion
  CeedCheck((P_1d > 0) && (Q_1d > 0), ceed, CEED_ERROR_INCOMPATIBLE, "CeedBasis dimensions are malformed");
```

Honestly not sure if these are needed parens to get the desired logic (ie. whether it just runs left-to-right or treats `<` expressions different than `&&` expressions), but doesn't hurt.